Kernel/Arch/aarch64/Registers.h

```diff
@@ -454,6 +454,54 @@ struct alignas(u64) SCR_EL3 {
 };
 static_assert(sizeof(SCR_EL3) == 8);
 
+// C5.2.18 SPSR_EL1, Saved Program Status Register (EL1)
+struct alignas(u64) SPSR_EL1 {
```

Comment:
this might render NZCV register unnecessary… is that the case?

Reply:
Yes, we don't actually need to store the NZCV flags separately, as they are already stored in the RegisterState in the spsr_el1 (Saved Program Status Register (EL1) field.